playbooks/setup.yml

```diff
@@ -16,7 +16,7 @@
     - role: create
     - { role: prereqs, when: "run_prereqs|bool == true and setup_minishift|bool == true" }
     - { role: minishift, when: "setup_minishift|bool == true" }
-    - { role: os_temps, when: setup_containers|bool == true }
+    - { role: k8s_objects }
     - { role: pipeline, when: setup_pipelines|bool == true }
```

Comment:
We don't need os_temps at all any longer with this change and shouldn't this have "set_containers|bool == true"

Reply:
Yes it should.